freespace2/freespace.cpp

```diff
@@ -947,6 +947,7 @@ void game_level_init()
 	NavSystem_Init();				// zero out the nav system
 
 	ai_level_init();				//	Call this before ship_init() because it reads ai.tbl.
+	multi_init_oo_and_ship_tracker();	// Inits/resets multiplayer ship tracking system.  Has to be done before creating any ships.
```

Comment:
There needs to be a multi check here. Might be best to put it in the function as I believe it's called from somewhere else too

Reply:
Yes, it's called once for the standalone code path and once for the regular code path.